docs/proposals/caching/README.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Caching member cluster resources for Karmada
+authors:
+reviewers:
+approvers:
+
+creation-date: 2022-02-14
+
+---
+
+# Caching member cluster resources for Karmada
+
+## Summary
+
+Provides a caching layer for Karmada to cache (member clusters) Kubernetes resources.
+
+## Motivation
+
+### Goals
+
+- Accelerate resource requests across regions
+- Provides a cross-cluster resource view
+- Compatible with (multi-cluster) multiple kubernetes resource versions
+- Uniform resource request entry
+- Reduce the pressure on the API server of the member cluster
+
+### Non-Goals
+
+## Proposal
+
+### User Stories (Optional)
+
+#### Story 1
+
+Imagine that we have a cross-regional application that needs to run in multiple regions (Beijing, Shanghai, Shenzhen) in order to provide service capabilities nearby. We achieve this by deploying a Kubernetes cluster in each region.
+
+Goals:
+
+- Get the distribution of this application in different clusters (resource aggregation)
+- (Quickly) get resource information of this application in different clusters (cross-regional resource access)
+
+#### Story 2
+
+Imagine we have a resource management platform, which consists of multiple microservices, each of which needs to frequently acquire, display, and operate resources from different clusters
+
+Goals:
+
+- Provide a (cross-cluster) operation entry for users (view resources, events, logs, and operate resources)
```

Comment:
What kind of `operations` we are talking about? 
It makes sense for me that `read` resources from the cache would relieve the pressure of the member cluster, but the `update/delete` operation doesn't as we need to operate the clusters anyway.